src/Build.OM.UnitTests/Construction/ProjectRootElement_Tests.cs

```diff
@@ -1853,6 +1854,10 @@ public void ReloadCanOverwriteUnsavedChanges()
         public void ReloadDoesNotLeakCachedXmlDocuments()
         {
             using var env = TestEnvironment.Create();
+            ChangeWaves.ResetStateForTests();
```

Comment:
It would be nice to have an example of what this test should look like after your change; otherwise, when removing the change wave, the default would be to eliminate this test entirely.

Reply:
Same for other tests